src/components/date-picker/date-picker.tsx

```diff
@@ -149,9 +148,6 @@ export class DatePicker {
     @Element()
     private host: HTMLLimelDatePickerElement;
 
-    @State()
-    private formattedValue: string;
```

Comment:
Yeah, it seems a lot more resonable to only format when rendering. Good change! 👍